applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -526,24 +526,83 @@ Use the following configuration options to customize the number of supported bri
+.. tabs::
 
-Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
-The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
-You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
-Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+   You have to consider several factors to decide how many Bluetooth LE bridged devices to support.
+   Every Bluetooth LE bridged device uses a separate Bluetooth LE connection, so you need to consider the number of supported Bluetooth LE connections (selected using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option) when you configure the number of devices.
+   Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
 
-.. parsed-literal::
-   :class: highlight
+   .. group-tab:: nRF53 DKs
 
-   west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
+      .. tabs::
 
-.. parsed-literal::
-   :class: highlight
+         .. group-tab:: Matter-Bridge over Wi-Fi configuration
+
+            Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
+            The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+            You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
+            Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+
+            .. parsed-literal::
+               :class: highlight
+
+               west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf" -DFILE_SUFFIX=nrf70ek
+
+         .. group-tab:: Matter-Bridge over Thread configuration
```

Comment:
You mentioned Matter Bridge over Thread only here and in the table below, but it is just a tab, so it is not visible enough. Shouldn't we inform about that somewhere at the bottom of this page, or somewhere else in our docs as well? I have a feeling that this is a bit of hidden information. @kkasperczyk-no what do you think?

Reply:
For example here: 
![image](https://github.com/user-attachments/assets/b55d4a41-2ca7-40ec-b57f-87ebfbd0536e)
there is mention that you can use only Wi-Fi as a transport for Matter in Matter Bridge.